apps/web-giddh/src/index.stage.html

```diff
@@ -58,6 +58,7 @@
                 console.error(error);
             }
         </script>
+        <script async src="https://cdn.headwayapp.co/widget.js"></script>
```

Comment:
Do not add new code when we already load the script tag in ngAfterViewInit.

Reply:
Check these all are unsed responsive code for small devices